tests/test_django.py

```diff
@@ -7,14 +7,17 @@
 import unittest
 from unittest import mock
 
-import django
-from django import test as django_test
-from django.conf import settings
-from django.contrib.auth.hashers import check_password
-from django.core.management import color
-from django.db import connections
-from django.db.models import signals
-from django.test import utils as django_test_utils
+try:
+    import django
+    from django import test as django_test
+    from django.conf import settings
+    from django.contrib.auth.hashers import check_password
+    from django.core.management import color
+    from django.db import connections
+    from django.db.models import signals
+    from django.test import utils as django_test_utils
+except ImportError:
+    raise unittest.SkipTest("django tests disabled.")
```

Comment:
Maybe reduce the try block. If django cannot be imported, it’s not available.
That avoids skipping the tests in case of an incorrect import in the form “from django.xxx import xxx”.
```suggestion
except ImportError:
    raise unittest.SkipTest("django tests disabled.")

from django import test as django_test
from django.conf import settings
from django.contrib.auth.hashers import check_password
from django.core.management import color
from django.db import connections
from django.db.models import signals
from django.test import utils as django_test_utils
```